bin/rmt-cli

```diff
@@ -1,29 +1,27 @@
 #!/usr/bin/env ruby
 
-require 'etc'
-
 rmt_path = File.expand_path('..', __dir__)
 require_relative '../config/boot'
 $LOAD_PATH.unshift File.join(rmt_path, 'lib')
 
+require 'etc'
```

Comment:
I think you don't need this here anymore since you moved that part to the `Utils`.

Reply:
It is still used to change user and group if rmt-cli is called as `root`. I didn't wrote a wrapper for those calls since they should only occur once in the project.